.bumpversion.toml

```diff
@@ -14,4 +14,4 @@ tag_message = "Bump version: {current_version} → {new_version}"
 allow_dirty = false
 commit = true
 message = "Bump version: {current_version} → {new_version}"
-commit_args = ""
+commit_args = ""
```

Comment:
these changes should not be committed since it makes it hard to review the code and they do not introduce any meaningful change.